content/influxdb/v1.8/tools/influx_inspect.md

```diff
@@ -458,7 +458,7 @@ Note: This can use a lot of memory.
 
 Use the `report-disk` command to review disk usage by shards and measurements for TSM files in a specified directory. Useful for determining disk usage for capacity planning and identifying which measurements or shards are using the most space.
 
-Calculates the total disk size (`total_tsm_size`) in bytes, the number of shards (`shards`), and the number of tsm files (`tsm_files`) for the entire directory, and the disk size (`size`) and the number of tsm files (`tsm_files`) for each shard. Use the `-detailed` flag to report disk usage (`size`) by database (`db`), retention policy (`rp`), and measurement (`measurement`).
+Calculates the total disk size (`total_tsm_size`) in bytes, the number of shards (`shards`), and the number of tsm files (`tsm_files`) for the specified directory. Also calculates the disk size (`size`) and number of tsm files (`tsm_files`) for each shard. Use the `-detailed` flag to report disk usage (`size`) by database (`db`), retention policy (`rp`), and measurement (`measurement`).
```

Comment:
This is good.

One more thing - it might be worth adding to the docs that for very small databases (i.e. collecting a single Telegraf for a day) this command may return no output, as TSM files are not created until the first WAL->TSM flush.

This prompted both Tim and I to briefly think the command was broken, I imagine our users might have the same reaction.

Reply:
Thank you, @lesam. Added note.